torch/distributed/rpc/__init__.py

```diff
@@ -21,7 +21,7 @@ def is_available():
     from .api import *  # noqa: F401
     import torch.distributed.autograd
 
-    def init_model_parallel(
+    def init_rpc(
         self_name,
         backend=backend_registry.BackendType.PROCESS_GROUP,
```

Comment:
We mention `model parallel` in the doc, should we rename that too?

Reply:
Yep, instead of " Initializes model parallel primitives" I will change it to " Initializes rpc  primitives"